Sources/Persistence/DiskMemoryPersistenceStack.swift

```diff
@@ -321,6 +380,16 @@ public final class DiskMemoryPersistenceStack: PersistenceStack {
     }
 }
 
+extension DiskMemoryPersistenceStack: NSCacheDelegate {
+
+    public func cache(_ cache: NSCache<AnyObject, AnyObject>, willEvictObject obj: Any) {
+
+        guard let data = obj as? Data else { return }
```

Comment:
why do we need this check? Additionally, we will only decrement the counter when it's `Data`

perhaps it could be an assertion (given we only deal with `Data` objects)?

Reply:
Because the delegate method sends Any.

Reply:
🤦‍♂️ 

perhaps we could still fire an assertion if we don't receive a `Data`, no?